QuizAppNew/View/HomeView.swift

```diff
@@ -59,9 +59,34 @@ struct HomeView: View {
                 Spacer(minLength: 0)
             }
             .background(Color.black.opacity(0.05).ignoresSafeArea())
+            .onAppear{
+                nazmul()
+                //load json data here. ....... 1st
+            
+            }
             .fullScreenCover(isPresented: $show, content: {
                 QAview()
             })
         }
     }
+    
+    func nazmul() {
+    
+        if let path = Bundle.main.path(forResource: "data", ofType: "json") {
+            do {
+                  let data = try Data(contentsOf: URL(fileURLWithPath: path), options: .mappedIfSafe)
+                let _posts = try? JSONDecoder().decode([QuestionModel].self, from: data)
```

Comment:
Why you set **try** with  **?**

Reply:
trying to execute the code.

Reply:
ki vul Baal bolo tomar kotha jodi na vuji keno use korcho ki docker chilo. tavole ami ar comment korbe na

Reply:
accha ami bujhiye bolets

Reply:
try? Mane hocche - Optional try? Is alternative way to convert throwing function. Jodi function ti successfully hoy tahole atir return value hobe optional value R jodi unsuccessful/fail hoy tahole nil return korbe .